crates/astria-proto/src/lib.rs

```diff
@@ -1,12 +1,15 @@
+pub use prost::{
+    DecodeError,
+    EncodeError,
+    Message,
+};
+
 #[allow(warnings)]
 #[allow(unreachable_pub)]
 #[rustfmt::skip]
 mod proto;
 
-pub use proto::generated::{
```

Comment:
Note here that with this commit the protos are no longer exported at the root level but only accessible through `generated`.